motr/client.h

```diff
@@ -1755,6 +1772,13 @@ struct m0_client_layout*
 m0_client_layout_alloc(enum m0_client_layout_type type);
 void m0_client_layout_free(struct m0_client_layout *layout);
 
+/* calculate protection info */
```

Comment:
Please use Doxygen format comments.

Please add comments for all parameters, return value.
Also, constrains, pre-, post-conditions.

Reply:
Done.